tests/app/shared/issue-tables/issue-sorter.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { getSortedData } from '../../../../src/app/shared/issue-tables/issue-sorter';
+import { MatSort } from '@angular/material';
+import { Issue } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_ASSIGNEES, ISSUE_WITH_EMPTY_DESCRIPTION, ISSUE_PENDING_MODERATION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+
+describe('issuer-sorter', () => {
+  describe('getSortedData()', () => {
+    const dummyTeam = new Team({
+      id: 'F09-2',
+      teamMembers: [],
+    });
+    const dummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+    const otherDummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_ASSIGNEES, dummyTeam);
+    const issuesList: Issue[] = [dummyIssue, otherDummyIssue];
+
+    const dummyTodoIssue: Issue = Issue.createPhaseModerationIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+    const otherDummTodoIssue: Issue = Issue.createPhaseModerationIssue(ISSUE_PENDING_MODERATION, dummyTeam);
```

Comment:
typo here `otherDummTodoIssue` --> `otherDummyTodoIssue` ?
Proposed renaming: `moderationIssue` and `otherModerationIssue`

Reply:
Yea, i think moderationIssue sounds better! 👍